test/image-sequencer.js

```diff
@@ -42,7 +42,7 @@ test('loadImages loads a DataURL image and creates a step.', function (t){
 
 if(!sequencer.options.inBrowser)
   test('loadImage loads an image from URL and creates a step. (NodeJS)', function (t){
-    sequencer.loadImage('URL','https://ccpandhare.github.io/image-sequencer/examples/red.jpg', function(){
+    sequencer.loadImage('URL','https://ccpandhare.github.io/image-sequencer/examples/images/red.jpg', function(){
```

Comment:
Yes, I had thought about this, too.
But I just couldn't come up with a substitute for this.
Maybe we could run this test if and only internet access is available?